### PR TITLE
modification of unix_setup.sh to fix installation problem

### PR DESCRIPTION
**Pull request by panagiotisntinis**
I tried to install/update autopsy on kali linux and I ran on some problems I solved. I tried to fix the easier of them with this pull request.
I tried to install autopsy on a directory that has a whitespace. For example:
`$ pwd`
`/home/axep/Download/to test`

I propose a fix for:
`cp $sleuthkit_jar_filepath $ext_jar_filepath`
to:
`cp $sleuthkit_jar_filepath "$ext_jar_filepath"`

I attach a screenshot to prove my point. I show my fix and then the problem I tried to fix
screenshot_20200428_2331142

Thanks for reviewing my request,
Panagiotis Ntinis

### CODE REVIEW

File: unix_setup.sh [hunk 1]
```diff
@@ -55,15 +55,15 @@ else
     exit 1
 fi
 
-ext_jar_filepath=$PWD/autopsy/modules/ext/sleuthkit-postgresql-$TSK_VERSION.jar;
+ext_jar_filepath="$PWD/autopsy/modules/ext/sleuthkit-postgresql-$TSK_VERSION.jar";
```
Comment (esaunders) [inline unix_setup.sh:58]: Not a big deal but you don't technically need to add the quotes here. The ext_jar_filepath variable gets set correctly with or without the quotes. It only needs to be wrapped in quotes when it is used in commands (rm and cp) below.

Comment (panagiotisntinis) [reply unix_setup.sh:58]: Thanks, for reviewing and for the comment.

Comment (esaunders) [reply unix_setup.sh:58]: You're welcome. Thanks for the submission :-)

**Comment by APriestman** [comment]
Sorry you need to revise one of the lines. The jar file has been renamed from "sleuthkit-postgresql-$TSK_VERSION.jar" to "sleuthkit-$TSK_VERSION.jar"

**Comment by panagiotisntinis** [comment]
I'm really new to github. I used "git push" command for my first time this week. I hope I have resolved the conflict.

**Comment by panagiotisntinis** [comment]
> Sorry you need to revise one of the lines. The jar file has been renamed from "sleuthkit-postgresql-$TSK_VERSION.jar" to "sleuthkit-$TSK_VERSION.jar"

I updated this line and it failed to pass AppVeyor test.
Then I pulled again from from sleuthkit/autopsy:develop to update my branch and it failled again.
Could you help me how to proceed?
Thanks

**Comment by esaunders** [comment]
@panagiotisntinis There are changes currently happening on the develop branch related to PostgreSQL that are causing builds to fail. I wouldn't worry about it in the context of your change since you have changed a runtime script rather than anything that would impact the build.

**Pull request merged**